pkg/services/host.go

```diff
@@ -175,8 +175,7 @@ func (hs *hostsService) InstallHost(ctx context.Context, h *model.Host, localNod
 	hs.UpdateHostStatus(ctx, h, model.UP)
 }
 
-//  ActivateHost activate  on the host
-// TODO: leaving it as public to allow a user activate a host
+// activates catapult-node on the host
```

Comment:
change to: "ActivateHost activates catapult-node on the host" to make the linter happy